.github/workflows/deploy.yaml

```diff
@@ -1,10 +1,7 @@
-# TODO use this trigger once tested
-# on:
-#   push:
-#     branches:
-#       - master
 on:
-  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
Whoops. thanks, these should obviously have been uncommented before that PR was merged.

Reply:
No problem!  All good now :-)